app/subcommands/service_subcommand.py

```diff
@@ -16,6 +17,12 @@ class ServiceSubcommand(Subcommand):
     _THREAD_NAME = None
 
     def run(self, *args, **kwargs):
+        # Set the multiprocessing method to 'spawn'. (The default on unix is 'fork'.) According to documentation, this
```

Comment:
Wow.  Good fix.  How are we ensuring that the set_start_method is only called once in the program?  What happens if it is called twice?  Is this testable?